### PR TITLE
Add npm develop script to watch for changes and run tests

### PR DESCRIPTION
**Pull request by gfpacheco**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
     "node": ">=4"
   },
   "scripts": {
-    "test": "./node_modules/.bin/mocha test --require ./test/globals.js"
+    "test": "mocha test --require ./test/globals.js",
```
Comment (thiagocaiubi) [inline package.json:9]: prefer locally mocha install to ensure it's version and availability

File: package.json [hunk 2]
```diff
@@ -6,7 +6,8 @@
...
+    "develop": "npm run test -- -w"
```
Comment (thiagocaiubi) [inline package.json]: `watch` could be a better name

File: package.json (hunk 1 above)
Comment (gfpacheco) [inline package.json:9]: Every command run through npm uses the local installation (if exists)

Comment (thiagocaiubi) [inline package.json:9]: 🆒 👍 I wasn't aware of it!

**Pull request merged**